modules/education/src/main/java/edu/stanford/spezi/modules/education/videos/component/ExpandableSection.kt

```diff
@@ -0,0 +1,196 @@
+package edu.stanford.spezi.modules.education.videos.component
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.expandVertically
+import androidx.compose.animation.shrinkVertically
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.ElevatedCard
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import edu.stanford.spezi.core.design.theme.Colors
+import edu.stanford.spezi.core.design.theme.Elevations
+import edu.stanford.spezi.core.design.theme.Shapes
+import edu.stanford.spezi.core.design.theme.Spacings
+import edu.stanford.spezi.core.design.theme.SpeziTheme
+import edu.stanford.spezi.core.design.theme.TextStyles
+import edu.stanford.spezi.core.design.theme.TextStyles.titleLarge
+import edu.stanford.spezi.modules.education.videos.Video
+import edu.stanford.spezi.modules.education.videos.VideoItem
+
+@Composable
```

Comment:
Composables have generic names here, but those are bound to video related screen only (also the parameters take video). I would indicate this in the function names and mark them internal. Please also review nullability of the parameters after adapting `Video` model

Reply:
good point. i originally intended it as a generic element too, but then more and more video logic was added and currently it doesn't seem like we want to reuse the composeable anywhere else, so i've now renamed it as you suggested and if it becomes relevant again later, i'll rebuild it to make it truly generic :)